### PR TITLE
fix test, de-graylist

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

File: tests/js/common/shell/shell-collection-cache-rocksdb.js [hunk 1]
```diff
@@ -209,20 +209,41 @@ function CollectionCacheSuite () {
         }
       });
 
-      c.unload(); // should destroy cache
-      f = c.figures(); // will load collection
+      f = c.figures();
+      print(f)
```
Comment (jsteemann) [inline tests/js/common/shell/shell-collection-cache-rocksdb.js]:
```suggestion
```

File: tests/js/common/shell/shell-collection-cache-rocksdb.js [hunk 2]
```diff
@@ -209,20 +209,41 @@ function CollectionCacheSuite () {
...
       assertTrue(f.cacheSize > 0);
+      assertTrue(f.cacheUsage > 0);
+      assertTrue(f.cacheLifeTimeHitRate > 0);
+
+      c.unload(); // should destroy cache
+      f = c.figures();
+      print(f)
```
Comment (jsteemann) [inline tests/js/common/shell/shell-collection-cache-rocksdb.js]:
```suggestion
```

File: tests/js/common/shell/shell-collection-cache-rocksdb.js [hunk 3]
```diff
@@ -209,20 +209,41 @@ function CollectionCacheSuite () {
...
+      assertEqual(f.cacheSize, 0);
       assertEqual(f.cacheUsage, 0);
-      assertEqual(f.cacheLifeTimeHitRate, 0);
+      assertFalse(f.hasOwnProperty('cacheLifeTimeHitRate'));
 
+      
       idxs = c.getIndexes(true);
+      print(idxs)
```
Comment (jsteemann) [inline tests/js/common/shell/shell-collection-cache-rocksdb.js]:
```suggestion
```

File: tests/js/common/shell/shell-collection-cache-rocksdb.js [hunk 4]
```diff
@@ -209,20 +209,41 @@ function CollectionCacheSuite () {
...
       idxs.forEach(function(idx, i) {
         if (idx.figures.cacheInUse) {
+          print(idx.figures)
```
Comment (jsteemann) [inline tests/js/common/shell/shell-collection-cache-rocksdb.js]:
```suggestion
```

File: tests/js/common/shell/shell-collection-cache-rocksdb.js [hunk 5]
```diff
@@ -209,20 +209,41 @@ function CollectionCacheSuite () {
...
           assertEqual(idx.figures.cacheLifeTimeHitRate, 0);
         }
       });
+
+      let doc; // load the collection again, put some cache hits...
+      for (let i=0; i < 10; i++) {
+        doc = c.exists("1");
+      }
+      f = c.figures();
+      print(f)
```
Comment (jsteemann) [inline tests/js/common/shell/shell-collection-cache-rocksdb.js]:
```suggestion
```

**Comment by dothebart** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/7083/

**Comment by dothebart** [comment]
http://jenkins.arangodb.biz:8080/job/TEST2-FC/31/ all blue.

**Pull request merged**